capstone/fitness/src/test/java/com/google/sps/EasySchedulerTest.java

```diff
@@ -0,0 +1,86 @@
+// // Copyright 2019 Google LLC
+// //
+// // Licensed under the Apache License, Version 2.0 (the "License");
+// // you may not use this file except in compliance with the License.
+// // You may obtain a copy of the License at
+// //
+// //     https://www.apache.org/licenses/LICENSE-2.0
+// //
+// // Unless required by applicable law or agreed to in writing, software
+// // distributed under the License is distributed on an "AS IS" BASIS,
+// // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// // See the License for the specific language governing permissions and
+// // limitations under the License.
+
+// package com.google.sps;
+// import com.google.sps.util.*;
+// import java.util.*;
+// import org.junit.*;
+// import static org.junit.Assert.*;
+
+
+// public class EasySchedulerTest { 
```

Comment:
Can you remove for this PR or leave a comment about why it's commented out?

Reply:
Irrelevant. Removed. Sorry about that.